.github/workflows/docs-site.yml

```diff
@@ -0,0 +1,19 @@
+name: docs-site
+
+on:
+  release:
+    types: [published]
+
+
+jobs:
+  publish-site:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Repository Dispatch
+        uses: peter-evans/repository-dispatch@v1
+        with:
+          token: ${{ secrets.REPO_ACCESS_TOKEN }}
+          repository: username/my-repo
```

Comment:
is this right?

Reply:
No, We have to setup details such as the PAT